harvard-lts/CURIOSity
# Fix the fts link generation

**Fix FTS Link Generation*
* * *

# What does this Pull Request do?
Fixes the issue in ticket: https://github.com/harvard-lts/CURIOSity/issues/298

# How should this be tested?

To see the current broken behavior, use the example on prod.

The on your local, pull this branch and rebuild. Do an FTS based search that is more than one word and see the fts link works (I did the same example VV did locally but will work in any collection). The do a single word to show that also still works.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
